ocrd/ocrd/cli/resmgr.py

```diff
@@ -39,23 +39,15 @@ def resmgr_cli():
     """
     initLogging()
 
-@resmgr_cli.command('discover')
-@click.option('-e', '--executable', 'glob', default='ocrd-*', help="Glob pattern for executables to inspect")
-@click.option('-n', '--dry-run', is_flag=True, default=False, help="Do not save to user resource_list.yml")
-def discover(glob, dry_run):
-    resmgr = OcrdResourceManager()
-    ocrd_processors = []
-    for executable, reslist in resmgr.discover(glob=glob, dry_run=dry_run):
-        print_resources(executable, reslist, resmgr)
-
 @resmgr_cli.command('list-available')
-@click.option('-e', '--executable', help='Show only resources for executable EXEC', metavar='EXEC')
-def list_available(executable=None):
+@click.option('-d/-D', '--dynamic/--no-dynamic', is_flag=True, default=True, help="Whether to look in processor --dump-json for resources")
```

Comment:
> We don't have such binary switches anywhere else in our CLIs. I recommend sticking to the negative switch only

But we do have `--overwrite/--no-overwrite` and  `--copy-files/no-copy-files`. I think it makes sense to enable users to make explicit calls, even if a default exists, precicesly because, like here, the default is changed.

Reply:
> We don't have such binary switches anywhere else in our CLIs. I recommend sticking to the negative switch only

But we do have `--overwrite/--no-overwrite` and  `--copy-files/no-copy-files`. I think it makes sense to enable users to make explicit calls, even if a default exists, precicesly because, like here, the default is changed.

Reply:
> But we do have `--overwrite/--no-overwrite`

Where would that be?

> and `--copy-files/no-copy-files`

I was not aware of that recent addition for `workspace.merge`. It seems to be the exception to the rule.

> I think it makes sense to enable users to make explicit calls, even if a default exists, precicesly because, like here, the default is changed

I disagree. It introduces ambiguity over what the default is – so without extra checking, I don't know what the default might be, and am forced to always specify a value just to be sure. If on the other hand, all options are one-sided, I can directly see where I would need to set it for the desired behaviour.

Changing the default would be the only good argument IMO – but where has that happened in OCR-D CLIs?

Reply:
> > But we do have `--overwrite/--no-overwrite`

Also in `ocrd workspace merge`

> > I think it makes sense to enable users to make explicit calls, even if a default exists, precicesly because, like here, the default is changed
> 
> I disagree. It introduces ambiguity over what the default is – so without extra checking, I don't know what the default might be, and am forced to always specify a value just to be sure. If on the other hand all options are one-sided, I can directly see where I would need to change something for the desired behaviour.
> 
> Changing a default value would be the only good argument IMO – but where has that happened in OCR-D CLIs yet?

I still think it's useful to allow switching calls from `--<feature>` to ``--no-<feature>` but if you insist, let's also remove it from `ocrd workspace merge` for consistency.

Reply:
> I still think it's useful to allow switching calls from `--<feature>` to ``--no-`but if you insist, let's also remove it from`ocrd workspace merge` for consistency.

Thanks!